File: dayFour/javaScript/afterbreak/question10.js
const scanner = require("prompt-sync")(); 
 

let sum = 0;
let average = 0;
  
    for( index = 1 ; index <=10; index ++){      
        let score = 0;
    console.log("Enter Score = ");
     if(score <= 100){
        sum +=score;
    average = sum/10
}
 }   

    console.log("The total sum of number is : ", sum);

    console.log ("The total avaerage is : ", average);

File: dayFour/python/afterbreak/question10.py
add = 0

average = 0

for number in range(1, 11):

    score = int(input ("Enter score : "))
    if( score <= 100):   

        add += score
        average = add/10;
print("The total average of the :",average)

print("The total sum of the :", add)
